_notes/why_must_i_learn_this.md

```diff
@@ -5,148 +5,108 @@ title: Why Must I Learn This?
-
-They do not spend extended periods of time applying the knowledge to solve real world problems.
-
-They do not have the chance to wrestle with all the fine nuances of how to manipulate that piece of knowledge.
-
-Therefore the appreciation for that knowledge is often not deep enough.
-
-The job of the school teacher is not to be the practitioner of the craft.
+Teachers are not positioned to do it well!
+They are not blessed with the opportunity to put these ideas into practice and see it through to execution.
+To wrestle with all aspects of the idea as it clashes head on with the messy nature of reality,
+and developing deep appreciation for the idea through that experience.
+That’s the job of a practitioner of the craft, not a school teacher.
 
 We know this.
+Business teachers invite successful founders to the classroom.
+Physics teachers invite astronauts to the classroom.
+Basketball coaches invite NBA players to the classroom.
+It’s to show that this knowledge is very much real and how it translates to real life.
 
-That’s why business teachers invite successful founders to the classroom.
-
-That’s why physics teachers invite astronauts to the classroom.
-
-That’s why basketball coaches invite NBA players to the classroom.
-
-The practitioner’s job is to showcase how the learnings translate to real world application.
-
-Hopefully, this brings relevance to the material they are studying.
+The practitioners are much better positioned than teachers for the second part.
+Their daily experience help showcase how the learnings translate to real world application,
+bringing relevance to the material they are studying,
+inspiring the students to study better.
 
-Hopefully, this inspires the students to study better.
 
 # Untapped potential
 
-Why can’t we have 
-
-- more engineers give coding workshops to students?
-- more marketers give copywriting masterclass to students?
-- more designers give art lessons to students?
-
-We believe that there’s a treasure trove of people in the industry who are passionate about their work and would love the chance to share that with the future generation.
-
-They are the ones best positioned to help students bridge what they learn to the real world.
-
-The best schools around the world already do this.
-
-Bringing role models into the classroom.
+What if engineers, doctors, designers, leaders were invited into the classrooms.
 
-Giving students the exposure they need.
+To share the knowledge they find interesting.
+To share the problems they are trying to solve
+To share the things that motivate them to learn more every day.
+Unconstrained by the same set of limitations which school teachers are bounded by.
 
-Now imagine if we are able to bring these industry experts to every willing student.
+Imagine how different class will feel like.
+Imagine the perspectives that they can bring to the students.
+Imagine the different answers they will receive if they were to ask “why must I learn this”.
```

Comment:
Imagine going home with not just an answer but more questions arising from asking “why must I learn this”

Reply:
Cant think of adding this in smoothly. There are quite a few concepts being brought up in a narrow length of text already.